test/functorch/test_vmap.py

```diff
@@ -3366,6 +3366,7 @@ def f(x):
             )(x)
             self.assertEqual(output, expected)
 
+    @skipIfTorchDynamo()
```

Comment:
All of these skips are because of dynamic shapes doesn't work well with eager backend, all passed if I switch to aot_eager or inductor backend.

Reply:
Why did this succeed before this PR? (I want to check that this PR isn't regressing any previous behavior)